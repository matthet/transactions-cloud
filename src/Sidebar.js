import React, { Component } from "react";
 
class Sidebar extends Component {
  render() {
    return (
      <div>
        <div className="subheading">{this.props.sideBarDetails.heading}</div>
        {this.props.sideBarDetails.data}
      </div>
    );
  }
}
 
export default Sidebar;