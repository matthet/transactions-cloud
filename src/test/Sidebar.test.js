import React from 'react';
import Sidebar from '../Sidebar';
import renderer from 'react-test-renderer';

test('Sidebar displays as expected', () => {
  const component = renderer.create(
    <Sidebar sideBarDetails={{heading: <div>DETAILS</div>}} />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});