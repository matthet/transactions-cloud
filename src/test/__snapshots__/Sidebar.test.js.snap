// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sidebar displays as expected 1`] = `
<div>
  <div
    className="subheading"
  >
    <div>
      DETAILS
    </div>
  </div>
</div>
`;
